reactioncommerce/reaction-component-library
# docs(#146): backfill docs with sections

Resolves #146
Impact: **minor**  
Type: **docs**

## Component
- Docs: Adds `### Overview` and `#### Usage` to all pages
- Docs: Reorganizes sidebar and adds a MiniCart section
- Chore: Changes the `addcomponent` script to make a new Markdown file with this template:
```
### Overview

#### Usage
```

## Screenshots
Docs look all better!

## Breaking changes
no

## Testing
1. run `docker-compose run --rm web node .reaction/scripts/addcomponent MyComponentNew`
2. Check the template to see if it matches:

```
### Overview

#### Usage
```
